### PR TITLE
[egl-registry, opengl-registry] Update egl-registry to 2020-02-20 and opengl-registry to 2020-03-25

### PR DESCRIPTION
**Pull request by Vincent-P**
**Describe the pull request**
It updates the egl-registry and opengl-registry to the latest commit available on master.
I also added some includes in opengl-registry, I took the lines from #9965.

### CODE REVIEW

**Comment by Vincent-P** [comment]
I am not sure why the CI fails on windows.

**Comment by Neumann-A** [comment]
https://github.com/microsoft/vcpkg/pull/9965#issuecomment-588860139

**Comment by PhoebeHui** [comment]
@Vincent-P, thanks for the PR!

Could you resolve the conflicts firstly?

**Comment by Vincent-P** [comment]
I resolved the conflicts, there are still problems with x86-windows.

File: ports/egl-registry/CONTROL [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Source: egl-registry
-Version: 2020-02-03
+Version: 2020-02-20
 Description: the EGL API and Extension Registry
```
Comment (NancyLi1013) [inline ports/egl-registry/CONTROL:3]: Could you please help add homepage here?
Homepage: 
`https://github.com/KhronosGroup/EGL-Registry`

File: ports/egl-registry/portfile.cmake [hunk 2]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/egl-registry/portfile.cmake:3]: `include(vcpkg_common_functions)` is no longer needed.
Could you please remove this line?

File: ports/egl-registry/portfile.cmake [hunk 3]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO KhronosGroup/EGL-Registry
-  REF aa9b63f3ab18aee92c95786a2478156430f809e4
-  SHA512 5ee7143c2cb46defbe1b2ecb0fabfb52fac2d9529b98c638dd4c04a312a62e7f7b3aee27d9749c92174ab967d533136b5881ce37ae9f2bee6685f52ffa8c8db6
+  REF 11478904448bbdf5757b798c856a525aa2b351b1
+  SHA512 f1e54810cb2948e9d8798d65507069bba4ee6534d719e792db11e36d600ef37e59a34262809d8b1e41160ae1e45a283fa322cd9d9a647985c48a6d7d6d1706ee
   HEAD_REF master
 )
```
Comment (NancyLi1013) [inline ports/egl-registry/portfile.cmake:10]: Could you please update 
```
file(
  COPY
    ${SOURCE_PATH}/api/egl.xml
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/egl-registry
)
file(
  INSTALL ${SOURCE_PATH}/sdk/docs/man/copyright.xml
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/egl-registry
  RENAME copyright
)
```
as
```
file(
  COPY
    ${SOURCE_PATH}/api/egl.xml
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT}
)
file(
  INSTALL ${SOURCE_PATH}/sdk/docs/man/copyright.xml
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT}
  RENAME copyright
)
```
?

File: ports/opengl-registry/CONTROL [hunk 4]
```diff
@@ -1,4 +1,4 @@
 Source: opengl-registry
-Version: 2020-02-03
+Version: 2020-03-25
 Build-Depends: egl-registry
 Description: the API and Extension registries for the OpenGL family APIs
```
Comment (NancyLi1013) [inline ports/opengl-registry/CONTROL:4]: Could you please add homepage here?
`Homepage: https://github.com/KhronosGroup/OpenGL-Registry`

File: ports/opengl-registry/portfile.cmake [hunk 5]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO KhronosGroup/OpenGL-Registry
-  REF 2223f5bebde4aa6b170fb32cdaaf580703fddb67
-  SHA512 4e9b570f242942bd45a6601a6b0fcf1dc265c6ba03acaf782a639e7399842dd7350c2d4876236df80a070b2bd9ce7cee88cf2d85f2c50cfba7878d1f9379bbe9
+  REF 4594c03239fb76580bc5d5a13acb2a8f563f0158
+  SHA512 c005a4eb7e5c17002647e7762ae1a7ecba0d0780a62d66f1afd3b7f45c1ca49bd5a069ab0fabb94de3ec971604586457932941fa8eb924cf5ac3a959d8f5f146
   HEAD_REF master
 )
```
Comment (NancyLi1013) [inline ports/opengl-registry/portfile.cmake:8]: Could you please update 
```
file(COPY
  ${SOURCE_PATH}/xml/gl.xml
  ${SOURCE_PATH}/xml/glx.xml
  ${SOURCE_PATH}/xml/wgl.xml
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/opengl-registry
)

file(
  INSTALL ${SOURCE_PATH}/xml/Makefile
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/opengl-registry
  RENAME copyright
)
```
as
```
file(COPY
  ${SOURCE_PATH}/xml/gl.xml
  ${SOURCE_PATH}/xml/glx.xml
  ${SOURCE_PATH}/xml/wgl.xml
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT}
)

file(
  INSTALL ${SOURCE_PATH}/xml/Makefile
  DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT}
  RENAME copyright
)
```
?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by Vincent-P** [comment]
I made all requested changes.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks for the update!

**Pull request merged**